### DISCUSSION TITLE
TransferRequestHandler for subsequent sitemap requesst when using a sitemap index file

### DISCUSSION THREAD

**Discussion by darrenshrwd** [Dotnet-Boxed/Templates#140] [Q&A]
When GetSitemapDocuments() creates a sitemap index file, it looks to me that the TransferRequestHandler below won't allow subsequent requests through beyond the index file. I shall experiment to see and then answer this one.

```
      <add name="SitemapXml" path="sitemap.xml" verb="GET" type="System.Web.Handlers.TransferRequestHandler" preCondition="integratedMode,runtimeVersionv4.0" />

```

**Comment by RehanSaeed** [comment]
What's the result of your experiment?

**Comment by darrenshrwd** [comment]
Ah all good as it creates urls like https://localhost:44301/sitemap.xml?index=1

(I didn't read it properly and thought it was creating urls like https://localhost:44301/sitemap1.xml)
